specification/glossary.md

```diff
@@ -74,6 +78,18 @@ An SDK-compliant secondary function that is abstracted by the Feature Flag API,
 
 Context object for flag evaluation, which may contain information about the runtime environment, details of the transport method encapsulating the flag evaluation, the host, the client, the subject (user), etc. This data may be used as a basis for differential evaluation of feature flags based on rules that can be defined in the flag system. Context data may be provided by merging static global context, arguments to flag evaluation, and implicit language-dependant state propagation mechanisms (thread-local storage, promise chains, continuations, etc).
 
+### Flag Evaluation
+
+The process of retrieving a feature flag value in it's entirety, including:
+
+- any effects resulting from hooks
+- resolving a flag value from a configured provider
+- falling back to a supplied default, in the case of abnormal execution
+
+### Flag Value Resolution
+
+The process of a provider retrieving a feature flag value from it's particular source-of-truth.
+
```

Comment:
I think this is a good distinction, but let me know if you think I'm being too pedantic. I think it could be a good basic for knowing how to name methods/functions.

Reply:
It's a spec. More things in the glossary are awesome. :D

Reply:
This is a bit nit-picky, but both Evaluation and Resolution suffer from being both a verb and a noun in English. To clarify that this is a verb, maybe we could use "Evaluating a Flag" and "Resolving a Flag Value" instead?

Reply:
This is exactly the kind of nit-pick I love.